__tests__/nc-games.test.js

```diff
@@ -85,4 +85,84 @@ describe("/api/reviews/:review_id", () => {
             }) 
         })
     })
+
+    describe("PATCH", () => {
+        test("Status 201: When a valid review_id is passed, if a positive number is passed, increment it by that much", () => {
+
+            return request(app)
+            .post("/api/reviews/2")
```

Comment:
Patch request please.

Reply:
That is my 4pm brain! Thanks for bringing that up, totally missed it! Will resubmit it shortly.